src/shared/hooks/useTable.test.ts

```diff
@@ -0,0 +1,66 @@
+import { renderHook, act } from '@testing-library/react';
+
+import { useTable } from './useTable';
+
+jest.mock('redux/modules', () => ({
+  workspaces: {
+    useData: () => ({ ownerId: 'mockOwnerId' }),
+  },
+}));
+
+describe('useTable hook tests', () => {
+  const mockAsyncFn = jest.fn();
+
+  test('useTable should return initial searchValue, order and page', () => {
```

Comment:
Each test runner logs 'description' of parent block and children blocks, so it's redundant to keep name of the parent block in each test, please remove the duplications `useTable` here and in other places.